pkgs/unified_analytics/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 3.0.1-wip
 
 - Enhanced `LogFileStats` data to include information about flutter channel counts and tool counts
+- Added new method to suppress telemetry collection temporarily for current invocation via `analytics.suppressTelemetry()`
```

Comment:
I think this should bump the minor version, since we're adding a new method.

Reply:
Wow you read my mind! :D i just pushed that update once you commented

Reply:
Well actually, is anyone implementing the `Analytics` class? If so, this is a breaking change, right?